aiohttp/web_response.py

```diff
@@ -33,6 +34,10 @@ class ContentCoding(enum.Enum):
 # HTTP Response classes
 ############################################################
 
+# Length in bytes of body which will trigger sync methods to process in a
+# thread pool to avoid blocking the main thread
+_BODY_LENGTH_THREAD_CUTOFF = 1024
```

Comment:
Could we have this configurable?

Reply:
where would you suggest this be settable?

Reply:
Perfect place would be Application, but real one could be Response. If you want custom tricky things you can construct Response object with proper settings. In the end, you only know which views returns big responses which could get any benefits from compression/json.dump in executor.

Reply:
made param